### PR TITLE
Monkey Clothing Part 7

### PR DESCRIPTION
**Pull request by DimWhat**
[Trivial]
## About the PR 
Gives monkeys unique backpack, belt, and radio sprites because there were sprite errors with them. Gives monkeys a few more dome style helmets as well! This PR also readds the ability for mutantrace offsets to some clothing items cause i thought they were unneccessary in my previous PR but it turns out they are good to keep around just in case..
image
image
image
image

~~NOTE: Currently trying to make TTV worn sprites properly work on chimps.~~ Fixed somehow :iiam:
## Why's this needed? 
yes.

## Changelog

```changelog
(u)DimWhat
(*)Added backpacks, belts, radio, and a couple special helmet sprites for monkeys!
```

### CODE REVIEW

**Comment by ZeWaka** [comment]
so is this still wip or wat

**Comment by DimWhat** [comment]
> so is this still wip or wat

Apologies, forgot to update the description when I fixed the TTV stuff.

File: code/mob/living/carbon/human/procs/update_icon.dm [hunk 1]
```diff
@@ -156,16 +156,15 @@
 	if (src.gloves)
 		wear_sanity_check(src.gloves)
 		var/icon_name = src.gloves.wear_state || src.gloves.item_state || src.gloves.icon_state
-
+		var/no_offset = 0
```
Comment (Zonespace27) [inline code/mob/living/carbon/human/procs/update_icon.dm:159]:
```suggestion
		var/no_offset = FALSE
```

File: code/mob/living/carbon/human/procs/update_icon.dm [hunk 2]
```diff
@@ -156,16 +156,15 @@
...
 		src.gloves.wear_image.layer = src.gloves.wear_layer
 
-		src.gloves.wear_image.pixel_x = 0
-		src.gloves.wear_image.pixel_y = 0
 
 		if (src.limbs && src.limbs.l_arm && src.limbs.l_arm.accepts_normal_human_overlays) //src.bioHolder && !src.bioHolder.HasEffect("robot_left_arm"))
 			if (islist(override_states) && ("glove-left_[icon_name]" in override_states)) //checking if the wearer is a mutant, and if so swaps the left glove with the special sprite if there is one.
 				src.gloves.wear_image.icon = src.mutantrace.clothing_icon_override
 				src.gloves.wear_image.icon_state = "glove-left_[icon_name]"
+				no_offset = 1
```
Comment (Zonespace27) [inline code/mob/living/carbon/human/procs/update_icon.dm:167]:
```suggestion
				no_offset = TRUE
```

File: code/mob/living/carbon/human/procs/update_icon.dm [hunk 3]
```diff
@@ -180,6 +179,7 @@
 			if (islist(override_states) && ("glove-right_[icon_name]" in override_states)) //above but right glove
 				src.gloves.wear_image.icon = src.mutantrace.clothing_icon_override
 				src.gloves.wear_image.icon_state = "glove-right_[icon_name]"
+				no_offset = 1
```
Comment (Zonespace27) [inline code/mob/living/carbon/human/procs/update_icon.dm:182]:
```suggestion
				no_offset = TRUE
```

File: code/mob/living/carbon/human/procs/update_icon.dm [hunk 4]
```diff
@@ -344,15 +348,24 @@
 	//tank transfer valve backpack's icon is handled in transfer_valve.dm
 	if (src.back)
 		wear_sanity_check(src.back)
-		src.back.wear_image.icon_state = src.back.wear_state || src.back.icon_state
-		src.back.wear_image.pixel_x = 0
-		src.back.wear_image.pixel_y = body_offset
-
+		var/wear_state = src.back.wear_state || src.back.icon_state
+		var/no_offset = 0
```
Comment (Zonespace27) [inline code/mob/living/carbon/human/procs/update_icon.dm:352]:
```suggestion
		var/no_offset = FALSE
```

File: code/mob/living/carbon/human/procs/update_icon.dm [hunk 5]
```diff
@@ -344,15 +348,24 @@
...
+		if (islist(override_states) && ("back-[wear_state]" in override_states)) //checks if they are a mutantrace with special back sprites and then replaces them if they do
+			src.back.wear_image.icon = src.mutantrace.clothing_icon_override
+			src.back.wear_image.icon_state = "back-[wear_state]"
+			no_offset = 1
```
Comment (Zonespace27) [inline code/mob/living/carbon/human/procs/update_icon.dm:356]:
```suggestion
			no_offset = TRUE
```

File: code/mob/living/carbon/human/procs/update_icon.dm [hunk 6]
```diff
@@ -369,15 +382,18 @@
 	if (src.glasses)
 		wear_sanity_check(src.glasses)
 		var/wear_state = src.glasses.wear_state || src.glasses.icon_state
+		var/no_offset
 		if (islist(override_states) && ("eyes-[wear_state]" in override_states)) //checks for special glasses sprites for mutantraces and replaces the sprite with it if there is one.
 			src.glasses.wear_image.icon = src.mutantrace.clothing_icon_override
 			src.glasses.wear_image.icon_state = "eyes-[wear_state]"
+			no_offset = 1
```
Comment (Zonespace27) [inline code/mob/living/carbon/human/procs/update_icon.dm:389]:
```suggestion
			no_offset = TRUE
```

File: code/mob/living/carbon/human/procs/update_icon.dm [hunk 7]
```diff
@@ -393,10 +409,19 @@
 	// Ears
 	if (src.ears)
 		wear_sanity_check(src.ears)
-		src.ears.wear_image.icon_state = src.ears.wear_state || src.ears.icon_state
+		var/no_offset = 0
```
Comment (Zonespace27) [inline code/mob/living/carbon/human/procs/update_icon.dm:412]:
```suggestion
		var/no_offset = FALSE
```

File: code/mob/living/carbon/human/procs/update_icon.dm [hunk 8]
```diff
@@ -393,10 +409,19 @@
...
+		var/wear_state = src.ears.wear_state || src.ears.icon_state
+		if (islist(override_states) && ("ears-[wear_state]" in override_states)) //checks if they are a mutantrace with special earwear sprites and then replaces them if they do
+			src.ears.wear_image.icon = src.mutantrace.clothing_icon_override
+			src.ears.wear_image.icon_state = "ears-[wear_state]"
+			no_offset = 1
```
Comment (Zonespace27) [inline code/mob/living/carbon/human/procs/update_icon.dm:417]:
```suggestion
			no_offset = TRUE
```

File: code/mob/living/carbon/human/procs/update_icon.dm [hunk 9]
```diff
@@ -494,9 +519,18 @@
...
 		wear_sanity_check(src.belt)
-		src.belt.wear_image.icon_state = src.belt.wear_state || src.belt.item_state || src.belt.icon_state
-		src.belt.wear_image.pixel_x = 0
-		src.belt.wear_image.pixel_y = body_offset
+		var/wear_state = src.belt.wear_state || src.belt.item_state || src.belt.icon_state
+		var/no_offset
+		if (islist(override_states) && ("belt-[wear_state]" in override_states)) //checks if they are a mutantrace with special belt sprites and then replaces them if they do
+			src.belt.wear_image.icon = src.mutantrace.clothing_icon_override
+			src.belt.wear_image.icon_state = "belt-[wear_state]"
+			no_offset = 1
```
Comment (Zonespace27) [inline code/mob/living/carbon/human/procs/update_icon.dm:527]:
```suggestion
			no_offset = TRUE
```

**Review by Zonespace27 — CHANGES_REQUESTED** [review]
Bools, aaaaa

**Pull request merged**